LibreHardwareMonitorLib/Hardware/Ring0.cs

```diff
@@ -222,6 +222,8 @@ public static void Open()
             {
 #if NETSTANDARD2_0
                 _isaBusMutex = new Mutex(false, isaMutexName);
+#elif NET5_0
```

Comment:
Can you merge these checks, this can be similar to the one in KernelDriver I think.